views/pages/toplevel/home.njk

```diff
@@ -4,6 +4,8 @@
 
 {% extends "../../layouts/main.njk" %}
 
+{% set bodyClass = "has-overlay" %}
```

Comment:
My only concern with this classname is that we also have the concept of an "overlay" (as opposed to `.overlay-text`), which is only used in a few places but is a modal-style box overlaid on top of the page when a successful "free material" order has been completed. This class sounds like it might be referring to that rather than the text for the hero? (see `overlay.scss` for more info!)

Reply:
Good shout, renamed this to the longer but hopefully more explicit `has-full-bleed-header`.